features/button.yml

```
@@ -1,5 +1,5 @@
 name: <button>
-description: The `<button>` element is an interactive element activated by a user gesture that performs an action such as submitting a form, or opening a dialog.
+description: The `<button>` HTML element represents a pushbutton that triggers some action, such as submitting a form or opening a dialog, styled as a labeled rectangular box by default.
```

`pushbutton` doesn't seem to be used in MDN or the spec. A quick search shows `push button` (with a space) to be used sometimes, eg shows up in W3Schools and various Q&A sites. Not a blocker, but seems overkill to bring a new term into use here. Would prefer to follow whatever the spec/MDN language.

---

Ah, I saw it in this table here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types. Yeah, we can just not do this.

```suggestion
description: The `<button>` HTML element represents a button that triggers some action, such as submitting a form or opening a dialog, styled as a labeled rectangular box by default.
```